docs/how-to/installation/docker.txt

```diff
@@ -56,29 +96,99 @@ master with the edited configuration:
 
 .. code::
 
-   docker run -v "$PWD"/master.yaml:/etc/determined/master.yaml determinedai/determined-master:VERSION
+   docker run \
+       -v "$PWD"/master.yaml:/etc/determined/master.yaml \
+       determinedai/determined-master:VERSION
+
+To start the master with environment variables instead of a configuration file:
+
+.. code::
+
+   docker run \
+       --name determined-master \
+       --network determined \
+       -p 127.0.0.1:8080:8080 \
```

Comment:
It prevents Docker from punching a hole in your firewall unexpectedly. This came from a doc I had for local development, where it definitely made sense; since this is for more general installation and the master might need to be accessible from outside, there should also be a note about doing it the other way in that case.

Reply:
changed to `8080:8080` and added a note on this below